opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -136,9 +136,9 @@ class Span(trace_api.Span):
     """
 
     # Initialize these lazily assuming most spans won't have them.
-    empty_attributes = BoundedDict(MAX_NUM_ATTRIBUTES)
-    empty_events = BoundedList(MAX_NUM_EVENTS)
-    empty_links = BoundedList(MAX_NUM_LINKS)
+    _empty_attributes = BoundedDict(MAX_NUM_ATTRIBUTES)
```

Comment:
was this to help the autodocumentation work better?

Reply:
Yes, it was just a detail I found. Without the underscore those identifiers are added to the documentation, I don't see any reason to have them there.